digital-twin/Samples/service/JdkSample/readme.md

```diff
@@ -0,0 +1,17 @@
+# Azure IoT Digital Twins Service SDK Samples
+
+**PREVIEW - WILL LIKELY HAVE BREAKING CHANGES**
+
+## Samples
+
+These samples are in order of complexity, and will teach you how to use each API within the digital twin service client
+
+[GetDigitalTwin](get-digital-twin) - Demonstrates how to check the state of a given digital twin
+
+[GetModel](get-model) - Demonstrates how to lookup the model definition for a given device capability model id, or interface id
+
+[InvokeCommand](invoke-digital-twin-command) - Demonstrates how to invoke a digital twin sync command on an active digital twin
+
+[InvokeAsyncCommand](invoke-digital-twin-async-command) - Demonstrates how to invoke a digital twin async command on an active digital twin
+
+[UpdateProperty](update-digital-twin) - Demonstrates how to update a single property on a digital twin
```

Comment:
Please updates all links in this page. they result in 404

Reply:
It works to me.